### PR TITLE
Feature/notes

### PR DESCRIPTION
**Pull request by PiZZAD0X**
Adds the ported notes module from grad_leaveNotes

- updated function structure, removed the ace overwrite functions
- initial optimizations

### CODE REVIEW

File: modules/modules.sqf [hunk 1]
```diff
@@ -49,6 +49,7 @@ Additional modules that can be enabled by removing the //
 //#include "in_game_brief\root.sqf"
 //#include "knock_down\root.sqf"
 //#include "marker_control\root.sqf"
+#include "notes\root.sqf"
```
Comment (EM-Creations) [inline modules/modules.sqf:52]: Do you want this as a default module?

File: modules/notes/functions/NOTES/fn_inspectNote.sqf [hunk 2]
```diff
@@ -0,0 +1,42 @@
+#include "script_component.hpp"
+
+private ["_handwriting"];
+params ["_handwriting"];
```
Comment (EM-Creations) [inline modules/notes/functions/NOTES/fn_inspectNote.sqf:4]: Are both these private and params _handwriting lines meant to be here?

File: modules/modules.sqf (hunk 1 above)
Comment (PiZZAD0X) [reply modules/modules.sqf:52]: No, was a mistake

File: modules/notes/script_component.hpp [hunk 3]
```diff
@@ -0,0 +1,2 @@
+#define COMPONENT GRAD_leaveNotes
```
Comment (EM-Creations) [inline modules/notes/script_component.hpp:1]: Does it matter that the component name differs from the `tag = ` decalaration in CfgFunctions? I always set them to be the same, but I dunno if that matters or not?

Comment (PiZZAD0X) [reply modules/notes/script_component.hpp:1]: should all be COMPONENT NOTES

File: modules/notes/functions/NOTES/fn_inspectNote.sqf (hunk 2 above)
Comment (EM-Creations) [reply modules/notes/functions/NOTES/fn_inspectNote.sqf:4]: @PiZZAD0X

**Review by EM-Creations — APPROVED** [review]

**Pull request merged**